luigi/tools/range.py

```diff
@@ -101,6 +101,9 @@ class RangeBase(luigi.WrapperTask):
     now = luigi.IntParameter(
         default=None,
         description="set to override current time. In seconds since epoch")
+    param_name = luigi.Parameter(
+        default=None,
```

Comment:
How would `positional=False` interact with `param_name`?  I don't understand its benefit.

`positional=False` + `param_name is not None` -> `task_cls(**{param_name: param})`
`positional=True` + `param_name is None` -> `task_cls(param)`
`positional=False` + `param_name is None` -> ?
`positional=True` + `param_name is not None` -> ?